packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -304,83 +307,66 @@ describe('ComboBox', function () {
   });
 
   it('can be disabled', async function () {
-    let {getByRole, queryByRole} = renderComboBox({isDisabled: true});
+    let tree = renderComboBox({isDisabled: true});
+    let {setInteractionType, setElement, getCombobox, getListbox, open} = testUtilUser.createTester('ComboBoxTester');
```

Comment:
I wanted to support destructing to mimic the testing library api and to make it a bit more ergonomic (aka no need to always do `cTester.something`). If test writer would like to go with the `cTester = testUtilUser` syntax that is still available so feels like the best of both worlds

Reply:
I've sat on it some more. I would like to push more for this to be the method I outlined above. The reason why is that this class has references to `this` inside of it, and it may have state. If we keep it all as class public methods accessed through the class, then that relationship is expected.

If we support the destructuring, people may be confused why calling any of the functions results in side effects to the other functions.

Reply:
I've updated the combobox util + tests, lemme know if that is along the lines of what you were expecting and I'll go ahead and do the rest of em

Reply:
👍🏻

Reply:
updated all other utils/tests to follow this now